docker-traceability-api/pom.xml

```diff
@@ -94,36 +65,6 @@
           <skipExistingHeaders>true</skipExistingHeaders>
         </configuration>
       </plugin>
-      <plugin>
-        <artifactId>maven-enforcer-plugin</artifactId>
```

Comment:
Very true, the enforcer settings in the parent pom are not the same. Will fix this.